core/src/main/scripts/importer/validateData.py

```diff
@@ -4196,6 +4465,12 @@ def main_validate(args):
 # ------------------------------------------------------------------------------
 # vamanos
 
+def _get_column_index(parts, name):
```

Comment:
The comment does not apply to this function. You should either define the function _above_ the comments like all other functions, or remove the comment.

Reply:
Comment removed.

Reply:
Half the comment is still there.

Reply:
I thought this served as a separator. I have removed the divider line.